### ISSUE TITLE
Map existing templates to new projects

### ISSUE THREAD

**Issue by ericdeflaminis** [env0/terraform-provider-env0#18]
This will tie into the current bug #13 with creating projects, but we would like the capability to tie org templates to newly created projects.

I would propose some new type of resource called something like "env0_project_template_attachment" that leverages the https://api.env0.com/blueprints/id/projects API endpoint to tie templates to projects

The TF code could look like:
```
data "env0_template" "aws-s3" {
  name = "aws-s3"
}

resource "env0_project" "project" {
  name = "My Project"
}

resource "env0_project_template_attachment" "attach" {
  template_id = data.env0_template.s3.id
  project_id = env0_project.project.id
}
```

**Comment by roni-frantchi** [comment]
Hey @ericdeflaminis !
Thanks for opening up this issue.  

Trying to prioritize this one - 
Do you plan to manage your Templates in the env0 webapp whilst making the project assignments in the provider?  
Or is it your intention to manage Templates via the provider as well?  
If it's the latter, we're currently working on supporting `import` in #35 - if that helps, we may start working on this issue a little later.

**Comment by ericdeflaminis** [comment]
@roni-frantchi thanks for taking a look. We plan to create projects through an outside process , and during that same process we would like to use the TF provider to assign existing templates to the project

**Comment by roni-frantchi** [comment]
Got it.  
So you do not plan to import Templates into back through the provider and manage those Templates via the provider at this point.  

We'll prioritize this one over import then.
Thanks!

**Comment by ericdeflaminis** [comment]
We may move our template management into a separate workspace as well yes, but a bit less of a priority for us.

**Comment by yaronya** [comment]
Fixed in #52
